src/main/java/com/fundation/webservice/model/PdfToImageConfig.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.webservice.model;
+
+/**
+ * Enumeration class that holds the default key values for the config parameter within the pdf to image conversion
+ *
+ * @author Alejandro Sánchez Luizaga
+ * @version 1.0
+ */
+public enum PdfToImageConfig {
+    dpi("150"),
```

Comment:
Maybe I think it sould be constant it looks magical number.

Reply:
is a value of an ENUM Yisus, it is not a constant.

Reply:
This a enumeration class intended to define default values and constant values as any other enumeration.